packages/core/lib/get-permissions.ts

```diff
@@ -20,6 +27,32 @@ export const getPermissions = ({
     config,
     globalPermissions,
   });
+  const changed =
```

Comment:
See below comment about additional params the resolvePermissions API, but we could use the `get-changed` lib here.